### PR TITLE
LUN-25: Add testflight delivery

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: ios/Lunes/Info.plist [hunk 1]
```diff
@@ -22,6 +22,8 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>ITSAppUsesNonExemptEncryption</key>
+    <false/>
```
Comment (klinzo) [inline ios/Lunes/Info.plist:26]: Not sure about this one

File: ios/Gemfile.lock [hunk 2]
```diff
@@ -240,7 +240,7 @@ GEM
     terminal-table (1.8.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
     thread_safe (0.3.6)
-    trailblazer-option (0.1.1)
+    trailblazer-option (0.1.0)
```
Comment (klinzo) [inline ios/Gemfile.lock:243]: Why did you downgrade?

**Review by klinzo — APPROVED** [review]

File: ios/Gemfile.lock (hunk 2 above)
Comment (SteffiStuffel) [reply ios/Gemfile.lock:243]: Error message due to wrong version. This way it works fine.

File: ios/Lunes/Info.plist (hunk 1 above)
Comment (SteffiStuffel) [reply ios/Lunes/Info.plist:26]: Added this, because otherwise the export complience form must be sent manually every time.

**Comment by SteffiStuffel** [comment]
@maxammann 
Would be nice if you can review. :)

File: ios/Lunes/Info.plist (hunk 1 above)
Comment (maxammann) [reply ios/Lunes/Info.plist:26]: Yes, this is correct.

**Review by maxammann — APPROVED** [review]

**Pull request merged**